common/src/main/java/io/netty/util/concurrent/SingleThreadEventExecutor.java

```diff
@@ -54,8 +56,15 @@ public void run() {
             // Do nothing.
         }
     };
+    private static final Runnable NOOP_TASK = new Runnable() {
```

Comment:
Perhaps we can just use the `WAKEUP_TASK` above? We skip `WAKEUP_TASK` in `pollTask()`, but `submit()` will wrap `WAKEUP_TASK` so it should just work.